Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -53,7 +53,7 @@ public static int ThreadMain(string[] args)
 
       if (exitValue == ExitValue.VERIFIED)
       {
-        exitValue = ProcessFiles(dafnyFiles, otherFiles.AsReadOnly(), reporter);
+        exitValue = ProcessFiles(dafnyFiles, otherFiles.AsReadOnly(), reporter); 
```

Comment:
We have some people introducing whitespace and some people removing it. Shouldn't there be a github hook? Anyone familiar with it? @seanmcl

Reply:
Sadly, we can't do it: https://stackoverflow.com/questions/22100581/github-server-side-git-hooks-i-e-pre-commit-and-post-commit

We could write a pre-commit hook, put it in this repo, and in the README ask contributors to install it in their git hooks directory before submitting.  wdyt?

Reply:
And there's this: https://pre-commit.com/